base_tier_validation/models/tier_validation.py

```diff
@@ -179,3 +180,9 @@ def request_validation(self):
                             })
                     # TODO: notify? post some msg in chatter?
         return created_trs
+
+    @api.multi
+    def restart_validation(self):
+        for rec in self:
+            if getattr(rec, self._state_field) in self._state_from:
+                rec.mapped('review_ids').sudo().unlink()
```

Comment:
Won't the sudo make the xmlrpc script open to vulnerability?

Reply:
actually is not even needed anymore. Access given to everyone on the security file.

Thanks @JayVora-SerpentCS